jha/src/views/enrolment/FPCareEligibilityPage.vue

```diff
@@ -50,24 +120,55 @@ export default {
       enrolmentRoutes.FPCARE_ELIGIBILITY_PAGE.path,
       enrolmentRoutes.FPCARE_ELIGIBILITY_PAGE.title
     );
+    const isApplyingForFPCare = this.$store.state.enrolmentModule.isApplyingForFPCare;
+    if (isApplyingForFPCare !== null) {
+      isApplyingForFPCare ? this.applyFPC = "Y" : this.applyFPC = "N";
+    }
+    this.meetsFPCCriteria = this.$store.state.enrolmentModule.meetsFPCCriteria;
+    this.hasFPCInfo = this.$store.state.enrolmentModule.hasFPCInfo;
+    this.studentMinorRefugee = this.$store.state.enrolmentModule.studentMinorRefugee;
+    this.hasDocuments = this.$store.state.enrolmentModule.hasDocuments;
+
+    this.radioOptionsApplyFPC = [
+      {
+        id: 'no',
+        label: 'No: (continue to MSP Supplementary Benefits).',
+        value: 'N',
+      },
+      {
+        id: 'yes',
+        label: 'Yes',
+        value: 'Y',
+      }
+    ];
 
     this.$nextTick(() => {
       this.isPageLoaded = true;
     })
   },
   validations() {
-    const validations = {};
+    const validations = {
+      applyFPC: {
+        validateQuestionsAnswered,
+      }
+    };
     return validations;
   },
   methods: {
     validateFields() {
+      this.saveData();
```

Comment:
I'm think we want to save the data to the Vuex store only if the validation passes. So this should be after the validation check.

Reply:
fixed... this was copied from Spouse info, so it should be changed there.